dd-bridge-android/src/main/kotlin/com/datadog/android/bridge/DdSdkConfiguration.kt

```diff
@@ -25,5 +26,6 @@ data class DdSdkConfiguration(
     val sampleRate: Double? = null,
     val site: String? = null,
     val trackingConsent: String? = null,
-    val additionalConfig: Map<String, Any?>? = null
+    val additionalConfig: Map<String, Any?>? = null,
+    val manualTracingEnabled: Boolean? = null
```

Comment:
this should come before `additionalConfig` ideally, but this is how it was declared in the API description. To fix with the next version.